main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/edgarduran/Go-Dispatch-Bootcamp/controllers"
+	"github.com/edgarduran/Go-Dispatch-Bootcamp/router"
+)
+
+func main() {
+	fmt.Printf("Hello\n")
+	c := controllers.New()
+	router := router.Setup(c)
+
+	fmt.Printf("Listening on port 3030\n")
```

Comment:
To have in mind, the use of `Printf` method is intended to be used with [printing verbs](https://pkg.go.dev/fmt#hdr-Printing); instead, you can use `Println`, and also get rid of that ugly `\n` at the end.

```suggestion
	fmt.Println("Listening on port :3030")
```